packages/storybook/stories/profilePicture.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { ProfilePicture, ProfilePictureGroup, sizeClasses } from '@dailydotdev/shared/src/components/ProfilePicture';
+
+// TODO: this story fails due to the below import which we will need to mock
+//       jest mocking is not currently supported in storybook
+// jest.mock('../hooks/useRequestProtocol', () => ({
+//   useRequestProtocol() {
+//     return {
+//       isCompanion: false,
+//     };
+//   },
+// }));
```

Comment:
I'm open to exploring mocking options here, but not sure if we should tackle it as a separate ticket?